financial-services-accelerator/accelerators/fs-is/repository/resources/wso2is-7.1.0-deployment.toml

```diff
@@ -500,6 +500,12 @@ json_path="/id_token/openbanking_intent_id/value" # Must be configured if "auth_
 handler="org.wso2.financial.services.accelerator.consent.mgt.extensions.manage.impl.DefaultConsentManageHandler"
 validator="org.wso2.financial.services.accelerator.consent.mgt.extensions.manage.impl.DefaultConsentManageValidator"
 
+[financial_services.consent.manage_extension]
+allowed_headers = [""]
+
+[financial_services.consent.authorize_extension]
+allowed_request_parameters = ["scope"]
```

Comment:
Here,it has introduced the config to filter attributes from the request object before passing to the pre-consent-authorize extension. But I feel we should remove this config ,coz;
1) We can just pass decoded request object without filtering,as it's anyway generated from third party,so spec developer can use the content of the request object to do custom validations
2) Adding more configs will confuse extension developer,as we have to give them a simple developer experience with less configs.

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/518/commits/22bb02c191d2d74199960e07e205b17b226074c7